src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -301,6 +301,20 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
+                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
code seems a bit duplicated though. anyway, I guess we don't have a way reduce..